core/src/apps/bitcoin/scripts_decred.py

```diff
@@ -11,7 +12,22 @@
     write_output_script_multisig,
     write_output_script_p2pkh,
 )
-from .writers import write_compact_size
+from .writers import (
+    write_bytes_fixed,
+    write_compact_size,
+    write_op_push,
+    write_bytes_fixed,
+    write_uint64_le,
+    op_push_length,
+)
+
+# These are decred specific opcodes related to staking.
+OP_SSTX = 0xBA
+OP_SSGEN = 0xBB
+OP_SSRTX = 0xBC
+OP_SSTXCHANGE = 0xBD
```

Comment:
please convert to micropython consts:
```
from micropython import const

_OP_SSTX = const(0xBA)
...
```
that way micropython bytecode compiler will do a search&replace instead of doing a lookup in the global namespace, and leading `_` stops the symbols from taking up space in the module dict

Reply:
I changed these as requested.